stdlib/3/multiprocessing/context.pyi

```diff
@@ -0,0 +1,168 @@
+    ) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to RawValue once a stub exists in multiprocessing.sharedctypes
+    def RawValue(self, typecode_or_type: Any, *args: Any) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to RawArray once a stub exists in multiprocessing.sharedctypes
+    def RawArray(self, typecode_or_type: Any, size_or_initializer: Union[int, Sequence[Any]]) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to Value once a stub exists in multiprocessing.sharedctypes
+    def Value(
+        self,
+        typecode_or_type: Any,
+        *args: Any,
+        lock: bool = ...
+    ) -> Any: ...
+    # TODO: typecode_or_type param is a ctype with a base class of _SimpleCData or array.typecode Need to figure out
+    # how to handle the ctype
+    # TODO: change return to Array once a stub exists in multiprocessing.sharedctypes
+    def Array(
+        self,
+        typecode_or_type: Any,
+        size_or_initializer: Union[int, Sequence[Any]],
+        *,
+        lock: bool = ...
+    ) -> Any: ...
+    def freeze_support(self) -> None: ...
+    def get_logger(self) -> Logger: ...
+    def log_to_stderr(self, level: Optional[str] = ...) -> Logger: ...
+    def allow_connection_pickling(self) -> None: ...
+    def set_executable(self, executable: str) -> None: ...
+    def set_forkserver_preload(self, module_names: List[str]) -> None: ...
+    def get_context(self, method: Optional[str] = ...) -> BaseContext: ...
+    def get_start_method(self, allow_none: bool = ...) -> str: ...
+    def set_start_method(self, method: Optional[str] = ...) -> None: ...
+    @property
+    def reducer(self) -> str: ...
+    @reducer.setter
+    def reducer(self, reduction: str) -> None: ...
+    def _check_available(self) -> None: ...
+
+class Process(object):
+    _start_method = ...  # type: Type[None]
+    @staticmethod
+    # TODO: type should be BaseProcess once a stub in multiprocessing.process exists
+    def _Popen(process_obj: Any) -> DefaultContext: ...
+
+class DefaultContext(object):
+    Process = ...  # type: Type[multiprocessing.Process]
+
+    def __init__(self, context: BaseContext) -> None: ...
+    def get_context(self, method: Optional[str] = ...) -> BaseContext: ...
+    def set_start_method(self, method: str, force: bool = ...) -> None: ...
+    def get_start_method(self, allow_none: bool = ...) -> str: ...
+    def get_all_start_methods(self) -> List[str]: ...
+
+if sys.platform != 'win32':
+    # TODO: type should be BaseProcess once a stub in multiprocessing.process exists
+    class ForkProcess(object):
```

Comment:
only one i was confused about was `_start_method = ...  # type: Type[None]` since by looking at PEP 526 it kind of looks like it needs to be _start_method = None  # type: str